### DISCUSSION TITLE
Announcing Publii v0.46.0 Release!

### DISCUSSION THREAD

**Discussion by bobmitro** [GetPublii/Publii#1608] [General]
Hi everyone!

We're excited to announce the release of Publii v0.46.0! This update brings several new features and improvements that many of you have been eagerly awaiting, including:

- Pages Support: Now you can create standalone pages with the same flexibility as posts, complete with nested page structures and customizable options.
- Enhanced Sync Mechanism: Improved file change detection to make synchronization faster and more reliable.
- Expanded Image Configuration: New options for managing images across different content types, including posts, tags, and authors.
- Typography Enhancements: Additional controls for font styles, including support for dedicated italic versions of variable fonts.
- Simple Theme v.3: Redesigned and rewritten, optimized for the latest Publii features.
- And much more!

You can find the full release notes [here](https://getpublii.com/blog/release-046.html).

Please use this thread to share your thoughts on the new release, report any bugs, or ask questions. Your feedback is invaluable in helping us improve Publii further!

Thank you for your continued support, and we hope you enjoy the new features in Publii v0.46.0!

**Comment by internettips** [comment]
Beyond outstanding -- thank you.

**Reply by rlesyk** [reply]
awesomeness and a game changer. I love it!!!

**Comment by gmmedia** [comment]
Great update.

**Comment by jpbraun** [comment]
Great update. I am using the Theme "Mellisa". Will there be an update for Public 0.46?

**Reply by bobmitro** [reply]
Forget about the MEllisa theme :), I will add it to the downloads section tomorrow.

**Reply by bobmitro** [reply]
Done! The Melissa theme is now available for download in the downloads section. I'm sorry for the earlier confusion.

**Reply by jpbraun** [reply]
Thanks Bob

**Comment by site-io** [comment]
Hi @bobmitro,

Thank you for the Publii-0.46.0 and Simple_3.0.0.0!

Before we switch to the Simple_3.0.0.0, it would be appreciated if you could please address errors and warnings from the following results:

https://demo.getpublii.eu/themes/simple/v3/
✅[  ->  ](https://validator.w3.org/nu/?showsource=yes&showoutline=yes&showimagereport=yes&checkerrorpages=yes&useragent=Validator.nu%2FLV+https%3A%2F%2Fvalidator.w3.org%2Fservices&acceptlanguage=&doc=https%3A%2F%2Fdemo.getpublii.eu%2Fthemes%2Fsimple%2Fv3%2F)
❌[  ->  ](https://jigsaw.w3.org/css-validator/validator?uri=https%3A%2F%2Fdemo.getpublii.eu%2Fthemes%2Fsimple%2Fv3%2F&profile=css3svg&usermedium=all&warning=2&vextwarning=&lang=en)

https://demo.getpublii.eu/themes/simple/v3/balance-blogging-and-traveling/
❌[  ->  ](https://validator.w3.org/nu/?showsource=yes&showoutline=yes&showimagereport=yes&checkerrorpages=yes&useragent=Validator.nu%2FLV+https%3A%2F%2Fvalidator.w3.org%2Fservices&acceptlanguage=&doc=https%3A%2F%2Fdemo.getpublii.eu%2Fthemes%2Fsimple%2Fv3%2Fbalance-blogging-and-traveling%2F)
❌[  ->  ](https://jigsaw.w3.org/css-validator/validator?uri=https%3A%2F%2Fdemo.getpublii.eu%2Fthemes%2Fsimple%2Fv3%2Fbalance-blogging-and-traveling%2F&profile=css3svg&usermedium=all&warning=2&vextwarning=&lang=en)

Thank you for your continued development!

**Reply by bobmitro** [reply]
really? It's a waste of time to engage with this kind of thread. Learn something about vWC3 validators, accessibility, etc., before posting such comments.

**Reply by site-io** [reply]
Hi @bobmitro,

Thank you for your response.

Thank you once again for the Publii-0.46.0 and Simple_3.0.0.0!

With Simple_3.0.0.0 we would like to get all 100 scores on [Lighthouse](https://developer.chrome.com/docs/lighthouse/overview) reports, like we are getting with Simple_2.8.3.0. And because of that, with Simple_2.8.3.0 we were able to outrank all competitors in SERPs!

It would be appreciated if you could please address errors and warnings from the results mentioned above.

Thank you once again for your continued development!

**Reply by bobmitro** [reply]
Please don't mislead people; familiarize yourself with these so-called errors. As for Lighthouse, it has excellent scores (with SEO excluded because the demo site is set to noindex):

<img width="903" alt="screenshot 2024-08-16 at 15 08 39" src="https://github.com/user-attachments/assets/64a79ad6-2f93-4503-a5cf-f771169c7ada">

**Comment by example-user30** [comment]
Congratulations on this new release!  Thank you for all your efforts and hard work.  Maybe this could have been 0.5!  It is such a milestone!

**Comment by joan-medina-roca** [comment]
This is great! Fantastic job, guys. Thanks for your time :D

**Comment by ghost** [comment]
Wow! What an awesome update. Thanks to everyone who contributed.

Up to now, "Pages" have been achieved by hiding posts and excluding them from the homepage. Having a tool to move existing posts to pages would be cool. I'm sure many of us have numerous blog posts that could be organized better as pages.

**Comment by michaelfluegge** [comment]
Hi, guys, I built a new website, can successfully test the connection but the upload got stuck when the process/the App switches to the message "Uploading website".  I use the identical settings as for my other website.
macOS Sonoma 14.6.1, Apple Silicon, 0.46.0 (build 16904), I have write access to the folder 

 Uploading exiting websites is no problem.

**Reply by michaelfluegge** [reply]
PS: I use FTP with SSL/TLS

**Reply by dziudek** [reply]
@michaelfluegge - first of all please check deployment-process.log and deployment-errors.log files under Tools -> Log viewer. Most probably upload process crashes due some unexpected problem

**Reply by michaelfluegge** [reply]
Thanks, @dziudek. I didn't find an error in the logs. But deleting and reinsert settings solved the problem.

**Comment by remiforall** [comment]
Hi,
Great update thanks !

First question : Is it possible to switch post to page and keeping beautiful url ?

Second question : Before updating Publii and Simple Theme, I had 100 in performance in lighthouse but since the update I just have 84% without changing anything else, any idea ?

**Reply by bobmitro** [reply]
There is no option for that; you have to create a page from scratch. 
The new Simple v.3, specifically the demo, achieves a performance score of 98% in Lighthouse. I recommend testing your website to identify what might be affecting your score. If the performance is lower, you may need to optimize your site further.

**Comment by nordseo** [comment]
Love the new page feature. Long awaited. One issue I'm having is images in pages aren't working. When uploaded to production, the image location shows my local machine file location. 

Do images for pages have to be manually uploaded into the media library?

Works fine for posts, however.

(Theme Mono 2.0.1.0)

**Reply by bobmitro** [reply]
What Publii version do you use? There was a problem in 0.46.0 but it was fixed in v. 0.46.1 and now it works very well.

**Reply by nordseo** [reply]
It works fine for the Block Editor on pages but not the WYSIWYG editor.

Using Version 0.46.0 (0.46.0)

**Reply by bobmitro** [reply]
Update to version 0.46.1. You will probably need to re-upload images.

**Reply by nordseo** [reply]
The update fixed it! Thanks! Version 0.46.1 (0.46.1)

**Comment by nordseo** [comment]
Quick questions about the new Repeater Field.

1. Can it be used on Posts (in postConfig)?
2. Does it have a search function?
3. Can it pre-filter based on a specific Tag or CustomField?



I'm attempting to select other posts to associate them with so they can be displayed in the post template.

**Reply by nordseo** [reply]
These custom fields cannot be used in postConfig?

 "type": "tags-dropdown"
  "type": "posts-dropdown"
  
  I'm looking for a way to display related posts in specific sections in the Post Template.
  
  Example:
  Post [Identity]
  
  Identity is associated with:
  [list of websites]
  [list of companies]
  
  If I create two custom post fields to select tags, then posts from that tag can be displayed in each section I assign them to in the post template.
  
  It just seems that custom fields should be available for posts, instead of the themeConfig.

**Reply by bobmitro** [reply]
All config options available for post/page are described in our dev [documentation](https://getpublii.com/dev/post-config-options/#what-other-config-options-are-availableandnbsplessbrgreater).

**Comment by dperarnaud** [comment]
Just to say THANKS.
Last 2 versions solved my problem when upload stop at 50 %.
Shinny new version is quick and stable !
Dominique

**Comment by tigexplorer** [comment]
Hi,

currently I'm using the Appimage Publii 0.45.2 with Terminal Theme 1.0.

What's the right way for updating a existing site?

Will the new Appimage V 46 come with the Terminal Theme V 2 and is my site `converted` by starting the new Publii Version?

Or have I do some things to update my site and the Theme manually in Publii 46?
